mewarpx/mewarpx/emission.py

```diff
@@ -1192,7 +1200,9 @@ def __init__(self, conductor, T, res_fac=5., transverse_fac=1.0, **kwargs):
             kwargs (dict): Any other keyword arguments supported by the parent
                 Emitter constructor (such as "emission_type").
         """
-        # Default Emitter initialization.
+        # Default initialization
+        if T is None:
+            T = conductor.T
```

Comment:
Would it be better to put this in the `Emitter` class? That allows `conductor=None`, so you would have to error if both `T` and `conductor` are set to `None`, but not setting `T` seems reasonable for other emitters that do use conductors, not just this one.